relm4-macros/src/widgets/gen/assign/assign_property.rs

```diff
@@ -100,31 +100,30 @@ impl AssignProperty {
                     }
```

Comment:
Why keep the `quote_spanned` in line 96? Is there something special there?

Reply:
That `quote_spanned` does not introduce an identifier into scope. `quote_spanned` invocations are fine as long as there are no new identifiers created inside.


This is problematic, as `__return` has a call site span.
```rust
quote_spanned! {Span2::call_site()=>
    let __name = #expr;
}
```

This isn't problematic as `quote_spanned` does not touch the span of substitutions:
```rust
quote_spanned! {Span2::call_site()=>
    let #name = #expr;
}
```

Instead of replacing `quote_spanned` with `quote`, I could have also just made the offending identifier a substitution with proper span.
```rust
let name = Ident::new("name", Span2::mixed_site());
quote_spanned! {Span2::call_site()=>
    let #name = #expr;
}